### PR TITLE
Fix look command

### PR DESCRIPTION
**Pull request by AndreasLETANTER**
Look command was having bad days for looking after item in south and west direction, because it was needed to reverse the tiles to seek...

Now fixed with more unit tests to ensure full work of the function

### CODE REVIEW

File: server/src/server_handling/events_handling/teams/commands/tools/handle_tiles_array.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    char **look_result = NULL;
+
+    for (unsigned int i = 0; i != level + 1; i++) {
+        nb_tiles += i * 2 + 1;
+    }
+    look_result = malloc(sizeof(char *) * (nb_tiles + 1));
+    for (unsigned int i = 0; i != nb_tiles; i++) {
+        look_result[i] = malloc(sizeof(char) * 100);
+        look_result[i][0] = '\0';
+    }
```
Comment (lbarreteau) [inline server/src/server_handling/events_handling/teams/commands/tools/handle_tiles_array.c:39]: Unverified malloc !! 
image

**Review by lbarreteau — CHANGES_REQUESTED** [review]

File: server/src/server_handling/events_handling/teams/commands/tools/handle_tiles_array.c (hunk 1 above)
Comment (AndreasLETANTER) [reply server/src/server_handling/events_handling/teams/commands/tools/handle_tiles_array.c:39]: Are you comparing me to 🤙🏿 ?

**Review by lbarreteau — APPROVED** [review]

File: server/src/server_handling/events_handling/teams/commands/look_command.c [hunk 2]
```diff
@@ -49,55 +49,29 @@ static unsigned int handle_map_outbound(int pos, unsigned int max_pos)
...
 client_t *client, unsigned int pos_x, unsigned int pos_y)
 {
-    char look_result[MAX_BUFFER_SIZE] = {0};
+    char **look_result = create_tiles_array(client->level);
     int y = pos_y;
     int level = client->level;
     unsigned int offset = offset_calculator[client->orientation];
 
-    for (int i = 0, x = pos_x - i; i != level + 1; i++) {
+    for (int i = 0, x = pos_x - i, nb_tiles = 0; i != level + 1; i++) {
```
Comment (HugoN1) [inline server/src/server_handling/events_handling/teams/commands/look_command.c:60]: unsigned int please

File: server/src/server_handling/events_handling/teams/commands/look_command.c [hunk 3]
```diff
@@ -49,55 +49,29 @@ static unsigned int handle_map_outbound(int pos, unsigned int max_pos)
...
         x = pos_x - i;
         for (int j = 0; j != i * 2 + 1; j++, x++) {
```
Comment (HugoN1) [inline server/src/server_handling/events_handling/teams/commands/look_command.c:62]: unsigned int please

**Review by HugoN1 — CHANGES_REQUESTED** [review]

**Review by HugoN1 — APPROVED** [review]

**Pull request merged**